apps/microsoft-teams/build-actions.js

```diff
@@ -5,6 +5,8 @@ const { hideBin } = require('yargs/helpers');
 
 const manifest = require('./contentful-app-manifest.json');
 
+require('dotenv').config({ path: './app-actions/.env' });
```

Comment:
Maybe an easier way (per my other comments above) would be to have a default .env template (instead of excluding it in git) that sets the env vars to be lame text strings and then we override with "real" values in the build steps? 🤔 

Since we're already relying on .env here I think we might as well.

Reply:
I updated this so that we have the .env with default text. For local development, you can set these to actual values. Then for CI, I created new contexts in Circle CI for apps staging and production and moved the environment variables there. This approach seems cleaner to me compared to the first iteration, but I'd love your feedback to see if this is a better direction!

Reply:
As discussed let's follow up later to require dotenv in tests, and also create a .env.development example file